### ISSUE TITLE
Subutai.io re-design

### ISSUE THREAD

**Issue by amol4anoff** [subutai-io/website#365]
1. We have dark blue site's color. But in the second section it is white - 

<img width="1879" alt="444" src="https://user-images.githubusercontent.com/33647491/35043910-92037be2-fbb8-11e7-98af-7ae3dbf1ea02.png">

Would be great to make it dark blue as the whole site

2. The down part before footer is black as well as footer - 

<img width="1879" alt="333" src="https://user-images.githubusercontent.com/33647491/35043975-cce48800-fbb8-11e7-89b3-dabbc8ead847.png">

Maybe change it to dark blue as the whole site concept, footer remains black.

3. That scrollbar is not clearly visible, when i firstly opened page, i thought it has only one first part with tribute to Dennis Ritchie - 

<img width="549" alt="555" src="https://user-images.githubusercontent.com/33647491/35044105-5783372c-fbb9-11e7-9f3d-a817fa27d34b.png">

So it would be cool to make it more visible and make the site color concept unified, as now it seems like 10 different designers worked on the site not advising with each other.

**Comment by efeefe** [comment]
"Would be great to make it dark blue as the whole site". Can you explain why that would be "great"?

**Comment by akarasulu** [comment]
@efeefe I think @amol4anoff clearly did by citing consistency.

**Comment by amol4anoff** [comment]
@efeefe Because now it looks like it was compiled from some different pieces - but the site usually has thematic colors, nothing spare. As it can be noticed there - the main color is dark blue - as it's prevailing upon others - white and black. So in my humble opinion, it would be good to make it just dark blue without white and black. But if you have any other ideas, that's awesome, man! Please share them.

**Comment by efeefe** [comment]
I wouldn't say that having a dark blue background in the header is inconsistent with white in the main content area. When scrolling down, it makes the page alternate between header, main content and footer in a very clear manner (see the [digitalocean](https://www.digitalocean.com/) website for example, or many many others). We have followed that when setting up the blog, as it makes sense anyway. Having the main content area's background in a dark color would make it harder to read - and I believe we want people to read, not only watch our material, right?

**Comment by eemilov2501** [comment]
@efeefe correctly noted that for more readability use white background with dark blue theme

**Comment by amol4anoff** [comment]
Anyway, it's up to you how it should be, i cannot impose my point of view.

**Comment by efeefe** [comment]
I do recognize the difference, but from my point of view it's a detail. Making the whole site dark blue and black won't improve it much, possibly the opposite. It's useful to have brighter parts on it.

**Comment by amol4anoff** [comment]
@efeefe yes, i agree that it will make sense regarding contrasts. Brighter parts will do some logics there, coz now its mixture -blue, black, white, dark blue looks like disorder.
